js/editor-libs/console.js

```diff
@@ -17,6 +17,22 @@
     };
 
     /**
+    * Formats output to indicate its type:
+    * - quotes around strings
+    * - square brackets around arrays
+    * @param {any} input - The output to log.
+    */
+    function indicateType(input) {
+        if (typeof(input) === "string") {
+            return `"${input}"`;
```

Comment:
Unfortunately template literals are not supported in IE[1] and we still need to support at least v11 :(
[1] https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
*sobs*